members/state_pa_gov_tom_wolf.yaml

```diff
@@ -1,53 +1,52 @@
 bioguide: state_pa_gov_tom_wolf
 contact_form:
   method: post
-  usechrome: true
   steps:
     - visit: "https://www.governor.pa.gov/contact/#OnlineForm"
     - find:
       - selector: "#newsletterModal2 button"
     - click_on:
       - selector: "#newsletterModal2 button"
     - find:
-      - selector: "#nf-field-3191"
+      - selector: div.nf-form-content input[type='text'][placeholder='First Name']
```

Comment:
Using the placeholder is pretty brittle, is there something stronger that can be used?

Reply:
We could use the `id`s. But those look auto-generated and could possibly change again

Reply:
yeah, that form does look pretty generated.. oh well 👍